src/DIRAC/Resources/Computing/HTCondorCEComputingElement.py

```diff
@@ -357,22 +350,24 @@ def killJob(self, jobIDList):
     #############################################################################
     def getCEStatus(self):
         """Method to return information on running and pending jobs."""
+        # Warning: information returned currently depends on the PilotManager and not HTCondor
+        # Results might be wrong if pilots or jobs are submitted directly via the CE.
```

Comment:
Alright.
"submitted directly via the CE" means "submitted manually calling `ce.submitJob(script)` instead of using a `SiteDirector`".
Long version: "if a user does not submit pilots via a `SiteDirector` and wants to get the CE Status, then the user has to manually add the pilots in `PilotAgentsDB` to get the right result, else the pilots will not be taken into account and the result will be wrong". Should I add this kind of sentence in the docstring? 
I will try to rephrase it.

Reply:
1. No (at least I don't think so) but the "CE should be able to work alone" as @fstagni said in a previous comment.
2. Ok done